packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -69,9 +69,9 @@ class VirtualListBasic extends Component {
 		 * ```
```

Comment:
I forgot now, but do you remember why we added the `delete rest.data;` above?

Reply:
I think we were passing `data` in `renderItem` in the past. I'll remove it.